### PR TITLE
Add tests for Tensor.* nn.* F.* docs

### PR DESCRIPTION
**Pull request by ssnl**
Test only for existence for now. I had to skip a lot of them so there a FIXME in the test.

Also I'm not testing torch.* because of namespace issue.

### CODE REVIEW

**Comment by pjh5** [comment]
This build pr/caffe2-py2-cuda9.0-cudnn7-setup-ubuntu16.04-build  was accidentally added to the PR trigger job. You may ignore it completely.

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -580,6 +574,16 @@ def add_docstr_all(method, docstr):
 In-place version of :meth:`~Tensor.cosh`
 """)
 
+add_docstr_all('cpu',
+               r"""
+cuda() -> Tensor
```
Comment (li-roy) [inline torch/_tensor_docs.py]: `cpu() -> Tensor`

File: test/test_torch.py [hunk 2]
```diff
@@ -146,6 +148,75 @@ def make_contiguous_slice(size, dtype):
...
+                       'sspaddmm',
+                       'storage_type',
+                       'tan',
+                       'to_dense',
+                       )
+        test_namespace(torch.nn)
+        test_namespace(torch.nn.functional, 'assert_int_or_pair', 'bilinear',
+                       'dropout', 'dropout2d', 'dropout3d', 'feature_alpha_dropout')
+        # TODO: add torch.* tests when we have proper namespacing on ATen functions
+        # test_namespace(torch)
```
Comment (li-roy) [inline test/test_torch.py]: just a question for my own clarification: do you mean that in the future we're gonna not have all ATen functions go to torch.*?

**Review by li-roy — APPROVED** [review]
Looks good. Just a couple comments.

**Pull request closed without merging**